README.md

```diff
@@ -132,28 +132,9 @@ Now, you have everything ready to start the installation of FOSSBilling,. Naviga
 ### Installing with Docker
 <a href="https://www.docker.com/"><img align="right" src="https://www.docker.com/wp-content/uploads/2022/03/horizontal-logo-monochromatic-white.png" alt="Docker logo" width="125"></a>
 
-This guide assumes you already have [Docker](https://docs.docker.com/get-docker/), [Git](https://git-scm.com) and [GNU make](https://www.gnu.org/software/make/) installed.
+We've created an official docker image, however, just as FOSSbilling is still under heavy development and testing, so is the docker image.
```

Comment:
I'd probably word it more like this:
We have developed an official Docker image, however, it should be noted that both the image and FOSSBilling are currently undergoing significant development and should not be considered production ready.

Regardless, you've entered the name as FOSSbilling rather than FOSSBilling, and a typo of our own name just cannot be a part of our repos.

Reply:
done